grazer/src/main/java/edu/asu/diging/grazer/core/conceptpower/impl/ConceptpowerConnector.java

```diff
@@ -17,9 +17,11 @@
 import edu.asu.diging.grazer.core.model.IConcept;
 
 @Service
-@PropertySource("classpath:config.properties")
```

Comment:
Is there a reason you're deleting this? I'm pretty sure you need this annotation to inject the conceptpower url?